lib/iris/tests/unit/mesh/components/test_MeshCoord.py

```diff
@@ -1097,3 +1097,9 @@ def test_updates_only_once(self, mocker):
         _ = self.meshcoord.update_from_mesh()
         _ = self.meshcoord.update_from_mesh()
         mocked.assert_called_once()
+
+        # Ensure it updates more than once if the mesh has been updated
+        # a second time
+        self.coord_on_mesh.points = np.ones(3)
+        _ = self.meshcoord.update_from_mesh()
+        mocked.assert_has_calls([mock.call(), mock.call()])
```

Comment:
It might be neater to write as `assert mocked.call_count == 2` (see https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_count)

Reply:
Now done :+1: